packages/webapp/src/middleware/voteWebsocket.js

```diff
@@ -12,10 +12,11 @@ import {
 
 
 // Define the Actions Intercepted by the Middleware
-const openVoteSocket = (payload) => ({type: 'voteSocket/open', payload});
-const closeVoteSocket = (payload) => ({type: 'voteSocket/close', payload});
-const sendVote = (payload) => ({type: 'voteSocket/send', payload});
-const markVoterInactive = (payload) => ({type: 'voteSocket/inactive', payload});
+const openVoteSocket = (payload) => ({ type: 'voteSocket/open', payload });
+const closeVoteSocket = (payload) => ({ type: 'voteSocket/close', payload });
+const sendVote = (payload) => ({ type: 'voteSocket/send', payload });
```

Comment:
Try to minimize all these formatting changes in the future if possible. It ends up creating a lot of diffs that aren't real. If needed, we can do a separate reformatting of the entire project with a linter.

Reply:
Yes, this is one of the things I wanted to do since there is no linter set up initially, and the codebase has some styling issues(missing spaces) that the IDE is trying to fix automatically.